src/lib/planviz/viz_utils.hpp

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+#include <string>
+#include <unordered_map>
+
+#include "dot_config.hpp"
+
+namespace opossum {
+
+extern const std::unordered_map<DotColor, std::string> dot_color_to_string;
```

Comment:
We are adding three files so that we can use `DotColor::Transparent` instead of `"transparent"`. What advantage do you see?

Reply:
Removed `viz_utils` to avoid a discussion here. Think I will insist on the dot_config module (also, because it will sooner or later be used by the Operator Visualizer as well.

Reply:
I'm fine with having a config for this. My question was more about the `DotColor::Transparent`. What advantage does `DotColor` give us over storing a string in the config? If we decide that we want the background to be yellow, we would have to add yellow to the enum, add it to the converter function and change it in the config. Also, it is not as if we are doing anything with that enum (comparing, iterating, ...).